rust/osm2lanes/src/transform/lanes_to_tags.rs

```diff
@@ -9,6 +9,7 @@ impl std::convert::From<DuplicateKeyError> for RoadError {
     }
 }
 
+#[non_exhaustive]
```

Comment:
I'm learning so much Rust by reviewing your PRs :) https://doc.rust-lang.org/reference/attributes/type_system.html#the-non_exhaustive-attribute

Reply:
I'm also learning, so don't assume I know what I am doing :P